File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/activitiesvb/vb/convertfaxtotask1.vb
' <snippetconvertfaxtotask1>



 ' Retrieve the fax.
 Dim retrievedFax As Fax = CType(_serviceProxy.Retrieve(Fax.EntityLogicalName, _faxId, New ColumnSet(True)), Fax)

 ' Create a task.
 Dim task As New Task() With {.Subject = "Follow Up: " &amp; retrievedFax.Subject, _
                              .ScheduledEnd = retrievedFax.CreatedOn.Value.AddDays(7)}
 _taskId = _serviceProxy.Create(task)

 ' Verify that the task has been created                    
 If _taskId <> Guid.Empty Then
     Console.WriteLine("Created a task for the fax: '{0}'.", task.Subject)
 End If

' </snippetconvertfaxtotask1>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/activities/cs/convertfaxtotask1.cs
// <snippetconvertfaxtotask1>



 // Retrieve the fax.
 Fax retrievedFax = (Fax)_serviceProxy.Retrieve(Fax.EntityLogicalName, _faxId, new ColumnSet(true));

 // Create a task.
 Task task = new Task()
 {
     Subject = "Follow Up: " + retrievedFax.Subject,
     ScheduledEnd = retrievedFax.CreatedOn.Value.AddDays(7),
 };
 _taskId = _serviceProxy.Create(task);

 // Verify that the task has been created                    
 if (_taskId != Guid.Empty)
 {
     Console.WriteLine("Created a task for the fax: '{0}'.", task.Subject);
 }

// </snippetconvertfaxtotask1>